src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -345,9 +345,10 @@ public String getActiveDeadlineSecondsStr() {
 
     public Map<String, String> getLabelsMap() {
         Set<LabelAtom> labelSet = getLabelSet();
-        ImmutableMap.Builder<String, String> builder = ImmutableMap.<String, String> builder();
+        ImmutableMap.Builder<String, String> builder = ImmutableMap.builder();
         if (!labelSet.isEmpty()) {
             for (LabelAtom label : labelSet) {
+                builder.put("jenkins/slave", "true");
```

Comment:
Are we expecting default labels as follow,  right?
```yaml
labels:
   jenkins : 'slave'
```

rather than 

```yaml
labels:
   jenkins/salve : true
```